__tests__/api.test.js

```diff
@@ -131,3 +131,57 @@ describe("/api/getReviewObjectById", () => {
       });
   });
 });
+
+////////////////////// PUT VOTES //////////////////////////////
+
+describe("/api/:review_id put request to change the vote on a comment", () => {
+  it("returns an error if nothing passed to the post request", () => {
+    return request(app)
+      .put("/api/reviews/1")
+      .send({})
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request format to update vote");
+      });
+  });
+
+  it.only("returns an error if id does not exist", () => {
+    return request(app)
+      .put("/api/reviews/100")
+      .send({ inc_votes: 1 })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("ID out of range");
+      });
+  });
+
+  it("returns an object of some type if passed an object", () => {
+    return request(app)
+      .put("/api/reviews/1")
+      .send({ inc_votes: 1 })
+      .expect(202)
```

Comment:
202 is accepted which is related to the accept header, 200 would be more appropriate here

Reply:
done